tests/integration/base_block_test.go

```diff
@@ -111,10 +110,30 @@ func runBlockE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite.
 	logger.Infof("Block Storage unmounted successfully")
 
 	logger.Infof("step 10: Deleting block storage")
-	_, dbErr := installer.BlockResourceOperation(k8sh, installer.GetBlockPoolStorageClassAndPvcDef(namespace, poolName, storageClassName, reclaimPolicy, blockName, "ReadWriteOnce"), "delete")
+	pvName, err := k8sh.GetPVCVolumeName(defaultNamespace, blockName)
+	pv, _ := k8sh.GetPV(pvName)
+	require.Nil(s.T(), err)
+	_, dbErr := installer.BlockResourceOperation(k8sh, installer.GetBlockPvcDef(blockName, storageClassName, "ReadWriteOnce"), "delete")
 	require.Nil(s.T(), dbErr)
-	require.True(s.T(), retryBlockImageCountCheck(helper, len(initBlockImages), 0, namespace), "Make sure a block is deleted")
-	logger.Infof("Block Storage deleted successfully")
+	if reclaimPolicy == "Delete" {
+		require.Equal(s.T(), fmt.Sprintf("%s", (*pv).Spec.PersistentVolumeReclaimPolicy), "Delete")
```

Comment:
i believe you could type cast to a string instead of needing the fmt.Sprintf()

Reply:
I tried casting to `string` before trying this approach. I will try to cast to string again and provide you with the error that I was getting.

Reply:
`string((*pv).Spec.PersistentVolumeReclaimPolicy)` seems to work.